bin/foreman_scap_client

```diff
@@ -46,22 +46,66 @@ ensure_scan_files () {
 }
 
 load_config () {
-  local config_file="/etc/foreman_scap_client/config"
+  local config_file="/etc/foreman_scap_client/config.yaml"
 
-  # Load config file
   if [ -f $config_file ]; then
-    # shellcheck disable=SC1090
-    source $config_file
+    eval "$(awk '
+    /^:server:/ {
+      print "SERVER=" $2;
+    }
+    /^:port:/ {
+      print "PORT=" $2;
+    }
+    /^:timeout:/ {
+      print "TIMEOUT=" $2;
+    }
+    /^:fetch_remote_resources:/ {
+      print "FETCH_REMOTE_RESOURCES=" $2;
+    }
+    /^:http_proxy_server:/ {
+      print "HTTP_PROXY_SERVER=" $2;
+    }
+    /^:http_proxy_port:/ {
+      print "HTTP_PROXY_PORT=" $2;
+    }
+    /^:ca_file:/ {
+      print "CA_FILE=" $2;
```

Comment:
In theory, all the paths may have spaces in them and not be quoted.

The following is valid yaml
```yaml
:ca_file: /an/example with/spaces
```